### PR TITLE
Subheaders with social media

### PR DESCRIPTION
**Pull request by Leehman**
Please review the 'subheaders'.

### CODE REVIEW

**Comment by Leehman** [comment]
snap 2018-01-29 at 10 50 29

**Comment by Leehman** [comment]
Image as requested.

File: examples/subheaders/Subheader.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                  <v-btn icon fab color="cyan">
+                    <v-icon>fa-instagram</v-icon>
+                  </v-btn>
+                </v-card-actions>
+              </v-card>
+            </v-flex>
+          </v-layout>
+          <v-subheader>Places to See </v-subheader>
+          <v-layout row wrap>
+            <v-spacer></v-spacer>
```
Comment (johnleider) [inline examples/subheaders/Subheader.vue]: Is this needed?

File: examples/subheaders/Subheader.vue [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                  </v-btn>
+                </v-card-actions>
+              </v-card>
+            </v-flex>
+          </v-layout>
+        </v-container>
+      </v-card>
+    </v-flex>
+  </v-layout>
+</template>
```
Comment (johnleider) [inline examples/subheaders/Subheader.vue]: line break between template and script

File: examples/subheaders/Subheader.vue [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      },
+      {
+        title: 'Good'
+      },
+      {
+        title: 'Finest'
+      }
+    ]
+  })
+}
```
Comment (johnleider) [inline examples/subheaders/Subheader.vue]: fix indentation of everything in here

File: examples/subheaders/Subheader.vue [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+            <v-icon>search</v-icon>
+          </v-btn>
+        </v-toolbar>
+        <v-container fluid grid-list-md class="grey lighten-4">
+          <v-subheader>Server &amp; Ops Management</v-subheader>
+          <v-layout row wrap>
+            <v-spacer></v-spacer>
+            <v-flex xs4 v-for="card in cards1" :key="card.title">
+              <v-card>
+                <v-card-media height="300px" :src="`https://picsum.photos/200/300?image=586`">
```
Comment (johnleider) [inline examples/subheaders/Subheader.vue]: If this is not dynamic, does not need to be a bound parameter

File: examples/subheaders/Subheader.vue [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+                  <span class="headline white--text  pl-3 pt-3" v-text="card.title"></span>
```
Comment (johnleider) [inline examples/subheaders/Subheader.vue]: extra space after white--text

File: examples/subheaders/Subheader.vue [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+          <v-toolbar-side-icon></v-toolbar-side-icon>
+          <v-toolbar-title>Grid List</v-toolbar-title>
+          <v-btn icon>
+            <v-icon>search</v-icon>
+          </v-btn>
+        </v-toolbar>
+        <v-container fluid grid-list-md class="grey lighten-4">
+          <v-subheader>Server &amp; Ops Management</v-subheader>
+          <v-layout row wrap>
+            <v-spacer></v-spacer>
```
Comment (johnleider) [inline examples/subheaders/Subheader.vue]: is this needed?

**Comment by bdeo** [comment]
Additionally, it's usually against company guidelines to use logos with modified colors. It might be a good idea to use the official colors for these logos.

**Comment by johnleider** [comment]
I agree with @bdeo 's feedback.

File: examples/subheaders/Subheader.vue (hunk 6 above)
Comment (Leehman) [reply examples/subheaders/Subheader.vue]: if this is in regards to v-spacer, no and its removed.

File: examples/subheaders/Subheader.vue (hunk 1 above)
Comment (Leehman) [reply examples/subheaders/Subheader.vue]: if this is in regards to v-spacer, no and its removed.

File: examples/subheaders/Subheader.vue (hunk 4 above)
Comment (Leehman) [reply examples/subheaders/Subheader.vue]: Fixed as requested.

File: examples/subheaders/Subheader.vue (hunk 5 above)
Comment (Leehman) [reply examples/subheaders/Subheader.vue]: Fixed as requested.

**Comment by Leehman** [comment]
snap 2018-01-29 at 14 08 33

**Comment by Leehman** [comment]
Switched ide to Visual Code. Changed icon text to white, Social media icons should reflect colors of brand, code modified as requested per JL, white space removed with ide component.

File: util/helpers.js [hunk 7]
```diff
@@ -32,3 +32,18 @@ export function isOnSale (variants) {
     return parseFloat(variant.price) < parseFloat(variant.compareAtPrice)
   })
 }
+
+export function randomNumber (min, max) {
+  return Math.floor(Math.random() * max) + min
```
Comment (jacekkarczmarczyk) [inline util/helpers.js]: `Math.floor(Math.random() * (max - min + 1)) + min`

**Pull request merged**